src/main/java/org/opentripplanner/middleware/bugsnag/BugsnagReporter.java

```diff
@@ -13,26 +16,36 @@
  */
 public class BugsnagReporter {
     private static Bugsnag bugsnag;
+    private static final Logger LOG = LoggerFactory.getLogger(BugsnagReporter.class);
 
     /**
-     * Initialize Bugsnag using API key when application is first loaded
+     * Initialize Bugsnag using the project notifier API key when the application is first loaded.
      */
-    private static void initializeBugsnag() {
+    public static void initializeBugsnagErrorReporting() {
         String apiKey = getConfigPropertyAsText("BUGSNAG_PROJECT_NOTIFIER_API_KEY");
         if (apiKey != null) {
             bugsnag = new Bugsnag(apiKey);
+        } else {
+            LOG.warn("Bugsnag project notifier API key not available. Bugsnag error reporting disabled.");
         }
     }
 
     /**
-     * Provide Bugsnag hook, if available, for reporting errors
+     * If Bugsnag has been configured, report error based on provided information.
      */
-    public static Bugsnag get() {
+    public static boolean reportErrorToBugsnag(Throwable throwable, String message) {
```

Comment:
I find it slightly more readable to have the string message be the first param (something about the longer string literal coming first or maybe because it mimics the Logger signature). Not a blocking change, but I think I would prefer that order.

Reply:
Updated.